.github/workflows/try-latest-pytorch.yml

```
@@ -126,6 +126,6 @@ jobs:
       mode: ${{ matrix.mode }}
       test_mode: accuracy
       dtype: ${{ matrix.dtype }}
-      models: "all"
+      models: "subset"
```

Nit: for consistency
```suggestion
      models: subset
```